airbyte-integrations/csv/src/main/resources/spec.json

```diff
@@ -0,0 +1,28 @@
+{
+  "destinationId": "8be1cf83-fde1-477f-a4ad-318d23c9f3c6",
```

Comment:
we will probably be stripping these ids out, but holding off on doing any of that until the remove integrations.java piece is done. will then confirm to whatever standard comes out of that.

Reply:
👍